iree/compiler/Conversion/LinalgToSPIRV/test/pipeline_test.mlir

```diff
@@ -0,0 +1,135 @@
+// RUN: iree-opt -split-input-file -pass-pipeline="iree-codegen-linalg-to-spirv-pipeline{use-vectorization}" %s | IreeFileCheck %s
+// RUN: iree-opt -split-input-file -pass-pipeline="iree-codegen-linalg-to-spirv-pipeline{use-vectorization use-workgroup-memory}" %s | IreeFileCheck %s
```

Comment:
Are they expected to produce same output? Don't we want to test specific output for both? If we only want to test no failure, maybe checking return code is good enough. Ie, remove IreeFileCheck.

Reply:
They have to produce the same number of `spv.CooperativeMatrixMulAddNV` instructions whether they use workgroup memory or not. I can check that the generated code is actually using workgroup memory in the second case, but its already checked in the pass specific test.

Reply:
Cool, then I think it's okay to leave it this way. Thanks for the explanation!